src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -359,4 +370,51 @@ private function getContainerEnvVars(ContainerBuilder $container): array
 
         return array_values($envs);
     }
+
+    protected function getUsagesForDefinition(Definition $definition): array
+    {
+        $class = $definition->getClass();
+
+        return null !== $class ? $this->getUsages($class) : [];
+    }
+
+    private function getUsages(string $class): array
+    {
+        return $this->usages[$class] ?? [];
+    }
+
+    private function buildUsageMapForContainer(ContainerBuilder $builder)
+    {
+        foreach ($builder->getDefinitions() as $id => $definition) {
+            $this->buildUsageMapForDefinition($builder, $id, $definition);
+        }
+    }
+
+    private function buildUsageMapForDefinition(ContainerBuilder $builder, string $id, Definition $definition)
+    {
+        foreach ($definition->getArguments() as $argument) {
```

Comment:
what's about factories and configurators? they could be references here in the inverse way

Reply:
Great suggestion, I’ll add this too!